### PR TITLE
feat: (app-shell) add a command to build and install app-shell locally

### PR DESCRIPTION
**Pull request by valorkin**

### CODE REVIEW

**Comment by fkolar** [comment]
@valorkin please rebase, you push the pr at the same time as I rebased from master

File: package.json [hunk 1]
```diff
@@ -75,7 +75,8 @@
     "build-storybook": "build-storybook",
     "nx": "nx",
     "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
-    "docker-test": "docker run -u $(id -u) --rm -v \"$PWD\":/app trion/ng-cli-karma npm run test"
+    "docker-test": "docker run -u $(id -u) --rm -v \"$PWD\":/app trion/ng-cli-karma npm run test",
+    "build-local-app-shell": "npm run build core && npm run build app-shell && npm run build:schematics && npm run sync-versions"
```
Comment (fkolar) [inline package.json:79]: @valorkin please rebase, and there are already commands to build app shell. I sent it the other day to slack

Comment (fkolar) [reply package.json:79]: please see the build-deploy-library

**Review by fkolar — CHANGES_REQUESTED** [review]

**Comment by valorkin** [comment]
> @valorkin please rebase, you push the pr at the same time as I rebased from master

it's was pushed 10 days ago )

File: package.json (hunk 1 above)
Comment (valorkin) [reply package.json:82]: this one is faster and simpler

Comment (fkolar) [reply package.json:82]: Do you really need it? The difference between standard command and this extra one is like 15 sec.

**Pull request merged**